experimental/Geometry/Misc.jl

```diff
@@ -0,0 +1,56 @@
+module Misc
+
+using Oscar
+
+export add_variables, coeffs_in_radical
+
+####################################################################
+#  
+#  Miscalleneous routines used in the above 
+#
+####################################################################
+
+
+function add_variables( R::MPolyRing, new_vars::Vector{String} )
```

Comment:
This one is very nitpicky, but: for future code, please don't use whitespace right of `(` and left of `)`. I.e.:
```suggestion
function add_variables(R::MPolyRing, new_vars::Vector{String})
```

Anyway, this is a minor thing. Perhaps we should just setup a default format setting for `JuliaFormatter`, and then ask everyone to use that for formatting their code... but it's really not important. So don't worry about this :-)